src/components/lobby/CreateMatchForm.js

```diff
@@ -61,13 +73,21 @@ const CreateMatchForm = ({ games, onCreate, disabled, loading }) => {
         value={players}
         onChange={(_, { value }) => setPlayers(value)}
       />
+      <Form.Input
+        fluid
+        label={`${t("create.match_name")} (${t("create.optional")})`}
+        value={name}
```

Comment:
I'd either pre-fill this with a generated name, or generate some name if user left this field blank, so we can avoid having random characters as room names

Reply:
I left the field empty by default, but made it required, so you need to click generate to fill it.
I think it's better this way, so users know where this name come from.

There is also a prompt message that instructs how to generate name